base/operators.jl

```diff
@@ -871,9 +871,37 @@ julia> fs = [
 julia> ∘(fs...)(3)
 3.0
 ```
+See also [Base.ComposedFunction](@ref).
 """
 function ∘ end
 
+"""
+    ComposedFunction{F,G} <: Function
```

Comment:
I think I've seen the convention to put the prefix in the docstring for non-exported symbols.

```suggestion
    Base.ComposedFunction{F,G} <: Function
```